src/main/java/it/pagopa/pn/kafka/bridge/middleware/kafka/deserializer/OnboardingSelfCareMessageDeserializer.java

```diff
@@ -1,22 +1,37 @@
 package it.pagopa.pn.kafka.bridge.middleware.kafka.deserializer;
 
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonDeserializer;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.deser.DeserializationProblemHandler;
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
 import it.pagopa.pn.kafka.bridge.model.OnboardingSelfCareMessage;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.kafka.common.serialization.Deserializer;
 
-import java.util.regex.Pattern;
+import java.io.IOException;
 
 @Slf4j
 public class OnboardingSelfCareMessageDeserializer implements Deserializer<OnboardingSelfCareMessage> {
 
-    private static final Pattern PROD_PN_PATTERN = Pattern.compile("product\" ?: ?\"prod-pn-");
     private final ObjectMapper objectMapper;
 
     public OnboardingSelfCareMessageDeserializer() {
         objectMapper = new ObjectMapper();
         objectMapper.registerModule(new JavaTimeModule());
+        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
La deserializzazione così non fallisce se si incontrano campi non previsti in OnboardingSelfCareMessage.

Reply:
Ok.